### DISCUSSION TITLE
Blazor Socket connection repeatedly disconneting after some period

### DISCUSSION THREAD

**Discussion by mandarshravankar** [elsa-workflows/elsa-core#4610] [Q&A]
Error: Connection disconnected with error 'Error: Server timeout elapsed without receiving a message from the server.'. 
<img width="898" alt="Capture" src="https://github.com/elsa-workflows/elsa-core/assets/30770408/eda3b223-ed1c-4a7c-9976-e8c61be1439d">

**Comment by abhjitsantra** [comment]
I am also facing similar issues. Sometimes the UI keeps on giving error message and reloading doesn't help.
